packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -163,6 +163,11 @@ export default class Proxy {
     )
     let date, proxyAuthenticationToken, xenstoreData
     try {
+      if (networkId !== undefined) {
+        await Promise.all(vm.VIFs.map(vif => xapi.deleteVif(vif)))
```

Comment:
On which network will the VIFs be if we don't delete them?

Reply:
Probably the XOA VIFs network if it exists.